src/cladetime/util/config.py

```diff
@@ -17,7 +17,7 @@ class Config:
     nextstrain_ncov_bucket = "nextstrain-data"
     nextstrain_ncov_metadata_key = "files/ncov/open/metadata_version.json"
     nextstrain_genome_metadata_key = "files/ncov/open/metadata.tsv.zst"
-    nextstrain_genome_sequence_key = "files/ncov/open/sequences.fasta.zst"
+    nextstrain_genome_sequence_key = "files/ncov/open/sequences.fasta.xz"
```

Comment:
ZSTD is a preferred compression format, but .xz file handling is built into Python, so switched to that

Reply:
(to be clear, it's entirely possible to handle ZSTD-compressed files in Python, I just didn't want to spend the time doing it when the lmza module makes it trivial to read .xz files; we can revisit if needed)